docs/source/getting_started/main_concepts.md

```diff
@@ -0,0 +1,124 @@
+# 🧠 Main Concepts
+
+## Overview
+Training a model with Streaming happens in 2 main steps:
+1. Convert your data to a StreamingDataset-compatible format
+2. Distributed model training with StreamingDataset
+
+Let's cover the key concepts in this process.
+
+## Dataset conversion
+Raw data samples need to be processed into a **stream**, or set of **shard** files, that allow for fast random access during training. Streaming supports the following file formats:
+* MDS (most performant)
+* XSV
+* JSONL
+
+A **shard** is a file, compatible with Streaming, that contains samples that are ready for training.
+
+A **stream** is a collection of shard files.
+
+The diagram below shows how raw data samples are converted to MDS shards using {class}`streaming.MDSWriter` objects.
+
+<img src="../_static/images/mds_writing.png" alt="MDS Writing" width="800"/>
+
+`MDSWriter` objects take in original dataset samples and convert them binary MDS shards, which contain serialized samples. The mapping from original files to shard files is not strict and can be one-to-one, many-to-one, or one-to-many. Each shard has a header that allows for fast random access to every sample during model training.
+
+As shown above, an `index.json` file is also created for the set of shard files, or stream, containing information such as the number of shards, number of samples per shard, shard sizes, etc.
+
+Below, we use `MDSWriter` to write out a stream to a remote location that contains integer columns `'x'` and `'y'` in each sample:
+<!--pytest.mark.skip-->
+```python
+columns = {'x': 'int', 'y': 'int'}
+output_dir = 's3://path/for/mds/dataset'
+with MDSWriter(out=output_dir, columns=columns, compression=compression, hashes=hashes, size_limit=limit) as out:
```

Comment:
I'm actually just going to take `compression`, `hashes`, and `size_limit` out since that level of detail isn't necessary in the high-level main concepts page